### PR TITLE
[SYCL] Fix self-build compilation after PR #9873

### PR DESCRIPTION
**Pull request by dm-vodopyanov**

### CODE REVIEW

File: .github/workflows/sycl_post_commit.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 name: SYCL Post Commit
 
+
 on:
-  push:
+  pull_request_target:
```
Comment (dm-vodopyanov) [inline .github/workflows/sycl_post_commit.yml:5]: @bader @aelovikov-intel, I changed this line to trigger post-commit checks on this PR, but all the checks look skipped, even lint. Is this a correct way to enable post-commit checks now, or probably it currently doesn't work as expected?

Comment (aelovikov-intel) [reply .github/workflows/sycl_post_commit.yml:5]: See https://github.com/intel/llvm/pull/10002 and https://github.com/intel/llvm/pull/10012.

Comment (dm-vodopyanov) [reply .github/workflows/sycl_post_commit.yml:5]: Thanks!

**Comment by dm-vodopyanov** [comment]
This patch tested for post-commit here: https://github.com/intel/llvm/pull/10020

**Review by steffenlarsen — APPROVED** [review]

**Comment by dm-vodopyanov** [comment]
@steffenlarsen, thanks! Could you please merge it without @intel/dpcpp-l0-pi-reviewers approval? The change in sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_device.cpp is obvious, it was just missed during code review for https://github.com/intel/llvm/pull/9873.

**Pull request merged**